### PR TITLE
DOC: rename add_geom to patch_stats

### PR DESCRIPTION
**Pull request by dgwakeman**
should address #1383

### CODE REVIEW

File: mne/source_space.py [hunk 1]
```diff
@@ -493,7 +493,7 @@ def _read_source_spaces_from_tree(fid, tree, add_geom=False, verbose=None):
 
 
 @verbose
-def read_source_spaces(fname, add_geom=False, verbose=None):
+def read_source_spaces(fname, patch_stats=False, verbose=None):
```
Comment (Eric89GXL) [inline mne/source_space.py]: We'll need to deprecate `add_geom` so we don't break people's code

File: mne/surface.py [hunk 2]
```diff
@@ -32,15 +32,15 @@
 # BEM
 
 @verbose
-def read_bem_surfaces(fname, add_geom=False, s_id=None, verbose=None):
+def read_bem_surfaces(fname, patch_stats=False, s_id=None, verbose=None):
```
Comment (Eric89GXL) [inline mne/surface.py]: Same here and if there are any other public-facing functions

**Comment by Eric89GXL** [comment]
Other than deprecation this LGTM

**Comment by dgwakeman** [comment]
how do I deprecate a variable? (example would be sufficient).

**Comment by Eric89GXL** [comment]
Basically we need to avoid breaking people's code whether they used `add_geom` in a set of either `args` or `kwargs`, so this should do it:
```
read_bem_surfaces(fname, patch_stats=False, s_id=None, verbose=None, add_geom=None):
...
    if add_geom is not None:
        warnings.warn('"add_geom" is deprecated and will be removed in "
                      "0.10, use "patch_stats" instead", DeprecationWarning)
        patch_stats = add_geom
```

File: mne/forward/forward.py [hunk 3]
```diff
@@ -441,7 +441,7 @@ def read_forward_solution(fname, force_fixed=False, surf_ori=False,
             raise ValueError('No parent MRI information in %s' % fname)
         parent_mri = parent_mri[0]
 
-        src = _read_source_spaces_from_tree(fid, tree, add_geom=False)
+        src = _read_source_spaces_from_tree(fid, tree, path_stats=False)
```
Comment (Eric89GXL) [inline mne/forward/forward.py]: as indicated by the CI failures, s/path_stats/patch_stats

Comment (dgwakeman) [inline mne/forward/forward.py]: yeah, just saw this I'll fix in the next round

File: mne/surface.py (hunk 2 above)
Comment (dgwakeman) [inline mne/surface.py]: here for example would it be:
``def read_bem_surfaces(fname, patch_stats=False, add_geom=None, s_id=None, verbose=None):``

and then the check you had.

Comment (agramfort) [inline mne/surface.py]: if should be

```
def read_bem_surfaces(fname, patch_stats=False, s_id=None, verbose=None, add_geom=None):

    if add_geom is not None:
          deprecation warning and set patch_stats to add_geom

```

Comment (Eric89GXL) [inline mne/surface.py]: @agramfort you didn't read the whole thread, did you? :)

Comment (agramfort) [inline mne/surface.py]: no… I try to keep up with limited internet access…

I'll let you handle this one then :)

**Comment by dgwakeman** [comment]
@Eric89GXL any other comments? Not sure what is up with Travis (it ran had one error: unrelated) and restarted. Does it always run twice?

**Comment by Eric89GXL** [comment]
Not usually, no. And FYI you can restart individual builds, which is better than restarting them all if only one fails.

**Comment by Eric89GXL** [comment]
LGTM, I'll merge once Travis is happy, feel free to ping me if you notice before I do

File: mne/source_space.py [hunk 4]
```diff
@@ -471,6 +473,10 @@ def _read_source_spaces_from_tree(fid, tree, add_geom=False, verbose=None):
     src : SourceSpaces
         The source spaces.
     """
+    if add_geom is not None:
+        patch_stats = add_geom
+        warnings.warn("`add_geom` is deprecated and will be removed in v1.0. "
+                      "Use `patch_stats instead.", DeprecationWarning)
```
Comment (agramfort) [inline mne/source_space.py]: `patch_stats -> `patch_stats`

File: mne/source_space.py (hunk 1 above)
Comment (agramfort) [inline mne/source_space.py]: +1

Comment (Eric89GXL) [inline mne/source_space.py]: Oh yeah, needs to be deprecated here as well

**Comment by Eric89GXL** [comment]
Minor style issues again. FYI you can run `make flake` locally to test it instead of waiting for Travis

**Comment by dgwakeman** [comment]
how about now @Eric89GXL ?

File: mne/source_space.py [hunk 5]
```diff
@@ -452,7 +453,8 @@ def _add_patch_info(s):
 
 
 @verbose
-def _read_source_spaces_from_tree(fid, tree, add_geom=False, verbose=None):
+def _read_source_spaces_from_tree(fid, tree, patch_stats=False, verbose=None,
+                                  ):
```
Comment (Eric89GXL) [inline mne/source_space.py]: it's better to move the last argument down, but I'll fix that in `master`

**Comment by Eric89GXL** [comment]
Great thanks @dgwakeman!

**Pull request merged**